ruby/private/bundle_fetch/gemfile_lock_parser.bzl

```diff
@@ -180,7 +180,7 @@ def parse_gemfile_lock(content):
                 version = version,
                 filename = "bundler-%s.gem" % version,
                 full_name = "bundler-%s" % version,
-                remote = "https://rubygems.org/",
+                remote = remote,
```

Comment:
I considered making this `remote or "https://rubygems.org/"`. Open to that if requested.

Reply:
Ah, I never finished this part, thank you.

Reply:
Actually, I wonder what is expected here? Should we use the last defined remote at all? This section only specifies bundler version.

Reply:
Right. it's a bit weird. The "right" solution may be to add an attribute to `rb_bundle_fetch` that describes the remote URL where to fetch the `bundler` gem. In practice, I suspect that 99% of users will be happy with this option since the `remote` inside their Gemfile.lock will serve `bundler` in addition to the specified gems. If you'd like the more explicit approach, happy to take that.

Reply:
If Gemfile.lock has multiple remotes, I believe it sorts them alphabetically so potentially this change might break in a situation when there is a remote after `rubygems.org`:

```
GEM
  remote: https://enterprise.contribsys.com/
...
GEM
  remote: https://rubygems.org/
```